test/setup/helpers.js

```diff
@@ -1,12 +1,23 @@
-export const spyify = (object, methodName, callback) => {
-  object['_original' + methodName] = object[methodName]
+const isSpy = (func) => {
+  return func.__spy
```

Comment:
great, thanks for making this test helper more robust! as far as style goes, i'd prefer a one-liner for such a simple function, and omit the parentheses around arguments when there's only one:
```js
const isSpy = func => !!func.__spy
```